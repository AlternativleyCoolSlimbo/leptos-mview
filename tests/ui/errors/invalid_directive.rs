use leptos_mview::view;

fn not_directive() {
    view! {
        div something:yes="b" {}
    };
}

fn not_class_name() {
    view! {
        div class:("abcd") = true {}
    };
}

fn not_style_name() {
    view! {
        div style:[1, 2]="black" {}
    };
}

fn not_event_name() {
    view! {
        button on:clicky-click={move |_| ()};
    };
}

fn main() {}
