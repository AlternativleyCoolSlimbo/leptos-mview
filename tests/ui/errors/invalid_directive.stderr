error: unknown directive `something`
 --> tests/ui/errors/invalid_directive.rs:5:13
  |
5 |         div something:yes="b" {}
  |             ^^^^^^^^^^^^^

error: expected identifier after `class:` directive
  --> tests/ui/errors/invalid_directive.rs:11:19
   |
11 |         div class:("abcd") = true {}
   |                   ^

error: expected identifier after `style:` directive
  --> tests/ui/errors/invalid_directive.rs:17:19
   |
17 |         div style:[1, 2]="black" {}
   |                   ^

error[E0425]: cannot find value `clicky_click` in module `leptos::ev`
  --> tests/ui/errors/invalid_directive.rs:23:19
   |
23 |         button on:clicky-click={move |_| ()};
   |                   ^^^^^^^^^^^^ not found in `leptos::ev`
